cpp/oneapi/dal/algo/pca/test/fixture.hpp

```diff
@@ -212,8 +212,18 @@ class pca_test : public te::crtp_algo_fixture<TestType, Derived> {
     }
 
     void check_infer_result(const pca::descriptor<Float, Method>& desc,
-                            const te::dataframe& data,
-                            const pca::infer_result<>& result) {}
+                            const pca::model<>& model,
+                            const table& data,
```

Comment:
Also, please add a TODO remark to change the definitions of all check functions with table as an argument instead of dataframe (dataframe used in train step checks)